<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWindow</class>
 <widget class="QMainWindow" name="QWMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>70</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDoc_new"/>
   <addaction name="actDoc_Open"/>
   <addaction name="actSave"/>
   <addaction name="actCloseAll"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="actSetFont"/>
   <addaction name="separator"/>
   <addaction name="actMDI"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actDoc_new">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建文档</string>
   </property>
   <property name="toolTip">
    <string>新建文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/200.bmp</normaloff>:/images/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/202.bmp</normaloff>:/images/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/204.bmp</normaloff>:/images/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actCloseAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/128.bmp</normaloff>:/images/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
   <property name="toolTip">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="actSetFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/506.bmp</normaloff>:/images/images/506.bmp</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actMDI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/230.bmp</normaloff>:/images/images/230.bmp</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
   <property name="toolTip">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/400.bmp</normaloff>:/images/images/400.bmp</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
   <property name="toolTip">
    <string>级联展开</string>
   </property>
  </action>
  <action name="actTile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/406.bmp</normaloff>:/images/images/406.bmp</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
   <property name="toolTip">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>QWMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
